collectors/python.d.plugin/alarms/alarms.chart.py

```diff
@@ -11,36 +11,47 @@
 disabled_by_default = True
 
 
-def charts_template(sm):
+def charts_template(sm, alarm_status_chart_type='line'):
     order = [
         'alarms',
+        'values'
     ]
 
     mappings = ', '.join(['{0}={1}'.format(k, v) for k, v in sm.items()])
     charts = {
         'alarms': {
-            'options': [None, 'Alarms ({0})'.format(mappings), 'status', 'alarms', 'alarms.status', 'line'],
+            'options': [None, 'Alarms ({0})'.format(mappings), 'status', 'alarms', 'alarms.status', alarm_status_chart_type],
```

Comment:
```suggestion
            'options': [None, 'Alarms ({0})'.format(mappings), 'status', 'status', 'alarms.status', alarm_status_chart_type],
```

`status` as the family looks better, do you agree?

<img width="136" alt="Screenshot 2021-03-04 at 13 59 26" src="https://user-images.githubusercontent.com/22274335/109954025-dd7f4480-7cf1-11eb-8d11-e5704e797694.png">